src/device/threads.rs

```diff
@@ -97,40 +109,57 @@ fn blkdev_monitor_thread(
+        move |should_stop: Arc<AtomicBool>| {
+            //std::thread::sleep(std::time::Duration::from_millis(10000));
+            loop {
+                match exit_condition(&blkdev) {
+                    Err(e) => warn!("cannot evaluate exit condition: {}", e),
+                    Ok(b) => {
+                        if b {
+                            return Ok(());
+                        }
                     }
                 }
-            }
-            {
-                let blkdev = try_with!(blkdev.lock(), "cannot unlock thread");
-                debug!("");
-                debug!("dev type {}", blkdev.device_type());
-                debug!("dev features b{:b}", blkdev.device_features());
-                debug!(
-                    "dev interrupt stat b{:b}",
-                    blkdev.interrupt_status().load(Ordering::Relaxed)
-                );
-                debug!("dev status b{:b}", blkdev.device_status());
-                debug!("dev config gen {}", blkdev.config_generation());
-                debug!(
-                    "dev selqueue max size {}",
-                    blkdev.selected_queue().map(Queue::max_size).unwrap()
-                );
-                debug!(
-                    "dev selqueue ready {}",
-                    blkdev.selected_queue().map(|q| q.ready).unwrap()
-                );
-            }
+                {
+                    let blkdev = try_with!(blkdev.lock(), "cannot unlock thread");
+                    // debug!("");
+                    // debug!("dev type {}", blkdev.device_type());
+                    // debug!("dev features b{:b}", blkdev.device_features());
+                    // debug!(
+                    //     "dev interrupt stat b{:b}",
+                    //     blkdev.interrupt_status().load(Ordering::Relaxed)
+                    // );
+                    // debug!("dev status b{:b}", blkdev.device_status());
+                    // debug!("dev config gen {}", blkdev.config_generation());
+                    // debug!(
+                    //     "dev queue {}: max size: {}, ready: {}, is valid: {}",
+                    //     //blkdev.selected_queue().map(Queue::max_size).unwrap()
+                    //     blkdev.queue_select(),
+                    //     blkdev.selected_queue().unwrap().max_size(),
+                    //     blkdev.selected_queue().unwrap().ready,
+                    //     blkdev.selected_queue().unwrap().is_valid() // crashes before activate()
+                    // );
+                    // debug!(
+                    //     "dev queue {}: avail idx {}, next avail idx {}",
+                    //     blkdev.queue_select(),
+                    //     blkdev.selected_queue().unwrap().avail_idx(Ordering::Relaxed).unwrap(),
+                    //     blkdev.selected_queue().unwrap().next_avail(),
+                    //     );
+                    debug!(
+                        "dev queue {}: irq status b{:b}",
```

Comment:
Was the code above removed on purpose?

Reply:
Yes. I have not found any use for this information yet. It just clogs the output. But i also don't want to delete it.